src/main/java/org/traccar/api/resource/PermissionsResource.java

```diff
@@ -17,7 +17,13 @@
 package org.traccar.api.resource;
 
 import java.sql.SQLException;
+import java.util.HashSet;
 import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
```

Comment:
Let's not use stream API. We are not using it yet and it's probably not really needed here.